### PR TITLE
Add float instructions to VM

### LINKED ISSUE #26: Float instructions
We should add float operations. `float` takes 4 bytes, so they can be kept in imm for now.

### PR DESCRIPTION
**Pull request by c71n93**
Closes #26

### CODE REVIEW

File: src/ChaiVM/interpreter/executor.cpp [hunk 1]
```diff
@@ -109,6 +108,86 @@ void Executor::divi(Instruction ins) {
         decoder::parse(codeManager_->getBytecode(regFile_.pc()));
     (this->*handlerArr[newIns.operation])(newIns);
 }
+void Executor::ldiaf(Instruction ins) {
+    double immd = static_cast<double>(std::bit_cast<float>(ins.immidiate));
+    regFile_.acc() = std::bit_cast<chsize_t>(immd);
+    advancePc();
+    Instruction newIns =
```
Comment (levBagryansky) [inline src/ChaiVM/interpreter/executor.cpp:115]: @c71n93 could you please replace these segments of code with function or macros

**Review by levBagryansky — CHANGES_REQUESTED** [review]
@c71n93 I think we can remove code duplication via macros or inline function

File: src/ChaiVM/interpreter/executor.cpp (hunk 1 above)
Comment (c71n93) [reply src/ChaiVM/interpreter/executor.cpp:115]: @levBagryansky how would you call this macro or function?

Comment (c71n93) [reply src/ChaiVM/interpreter/executor.cpp]: @levBagryansky done

**Pull request merged**